share/wrf_timeseries.F

```diff
@@ -645,6 +671,23 @@ SUBROUTINE write_ts( grid )
          END DO
          CLOSE(UNIT=iunit)
 
+         iunit = get_unused_unit()
+            IF ( iunit <= 0 ) THEN
+            CALL wrf_error_fatal('Error in write_ts: could not find a free Fortran unit.')
+            END IF
```

Comment:
The indentation here looks off.

Reply:
I just noticed that we're following what was done for other profiles. I suppose we can keep this as is, or fix the indentation in the corresponding code for other profiles.